modules/giapi/src/main/scala/giapi/client/gpi/GpiClient.scala

```diff
@@ -98,22 +110,67 @@ final class GpiClient[F[_]](override val giapi: Giapi[F]) extends GiapiClient[F]
       ),
       DefaultCommandTimeout
     )
+
+  override def genericApply(configuration: Configuration): F[CommandResult] = {
+    // TODO Implement a smarter apply
```

Comment:
Here we'll eventually make the apply smarter by only sending configs when needed